controlplane/.env.example

```
@@ -74,3 +74,11 @@ PROMETHEUS_ENABLED="false"
 PROMETHEUS_HTTP_PATH="/metrics"
 PROMETHEUS_PORT="8088"
 PROMETHEUS_HOST="localhost"
+
+# Sentry integration
+SENTRY_ENABLED="false"
+SENTRY_DSN=""
+SENTRY_SEND_DEFAULT_PII="false"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)